xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererMediaCodec.cpp

```diff
@@ -116,8 +116,8 @@ CRenderInfo CRendererMediaCodec::GetRenderInfo()
 {
   CRenderInfo info;
   info.formats = m_formats;
-  info.max_buffer_size = NUM_BUFFERS;
-  info.optimal_buffer_size = 2;
+  info.max_buffer_size = 4;
```

Comment:
> RenderManager won't skip a frame immediately if it detects a lateness of less than frametime

Mmm... Call me thick, but I don't see immediately in what having an additional buffer would influence that...
Or do you need at least fps/Hz*2 buffers? What about 60fps on 24Hz, then...